js/copy-service/CopyService.ts

```diff
@@ -140,20 +136,24 @@ class CopyService {
   /**
    * Returns the current merged set of of registered copy. This is helpful to get the current
    * set of registered copy when copy is registered via many sources.
-   *
-   * @param {AST} _node [PRIVATE] Node of the registered copy to get keys from when recursing.
-   * @returns {object} The registered copy, in un-parsed form.
    */
-  getRegisteredCopy(_node = null) {
-    const tree = {};
+  getRegisteredCopy(
+    passedNode?: RegisteredCopyNode
+  ): { [key: string]: RegisteredCopyNode; } {
+    const tree: { [key: string]: RegisteredCopyNode; } = {};
+    const nodeToIterate = passedNode || this._registeredCopy;
+
+    if (!_.isPlainObject(nodeToIterate)) {
```

Comment:
Can we remove the duplicative `_isPlainObject` check below?

Reply:
This line checks `passedNode || this._registeredCopy` whereas line 157 checks `node`, which is a child of the plain object `nodeToIterate`.

i.e.

```ts
{ // line 146 check
  'some': {}, // line 157 check
  'example': 'copy' // line 157 check
}

```